contracts/validator-manager/PoSValidatorManager.sol

```diff
@@ -66,22 +66,24 @@ abstract contract PoSValidatorManager is
 
     // keccak256(abi.encode(uint256(keccak256("avalanche-icm.storage.PoSValidatorManager")) - 1)) & ~bytes32(uint256(0xff));
     // TODO: Unit test for storage slot and update slot
-    bytes32 private constant _POS_VALIDATOR_MANAGER_STORAGE_LOCATION =
+    bytes32 public constant POS_VALIDATOR_MANAGER_STORAGE_LOCATION =
```

Comment:
Why did you change this to be public? Since we can't enforce access within the struct definition, my strong preference would be to make the storage private and expose public getters and setters as needed.

Reply:
made them public to be used in unit test for storage slots. These variables are constants so do not need to worry about derived contracts accidentally accessing and overriding the value.

Reply:
Fair point 👍